types/vis/vis-tests.ts

```diff
@@ -54,7 +57,7 @@ console.log('item1', item1);
 
 // retrieve a filtered subset of the data
 let items = data.get({
-  filter: (item) => {
+  filter: (item: any) => {
```

Comment:
do not use any

Reply:
This in fact is not related with this pull request, but hooks (which have rules newer than latest vis change) are failing because implicit type warning. This is just to make hooks happy, I really dont think this should be a blocker for this PR because in fact i dont know what should be there.